src/gui/transferlistsortmodel.cpp

```diff
@@ -165,6 +165,9 @@ int TransferListSortModel::compare(const QModelIndex &left, const QModelIndex &r
     case TransferListModel::TR_CATEGORY:
     case TransferListModel::TR_NAME:
     case TransferListModel::TR_SAVE_PATH:
+    case TransferListModel::TR_DOWNLOAD_PATH:
+    case TransferListModel::TR_INFOHASH_V1:
+    case TransferListModel::TR_INFOHASH_V2:
```

Comment:
Info hash shouldn't use natural comparison.
Usual string compare should be sufficient.